src/linalg/spectral.jl

```diff
@@ -59,7 +59,7 @@ function _adjacency_matrix(g::AbstractGraph, T::DataType, neighborfn::Function,
                 nz -= 1
             end
         end
-        dsts = sort(collect(neighborfn(g, j))) # TODO for most graphs it might not be necessary to sort
+        dsts = sort!(collect(neighborfn(g, j))) # TODO for most graphs it might not be necessary to sort
```

Comment:
In place `sort!` can be algorithmically a fair bit slower than the out of place version.

Reply:
From my benchmark, it seemed to be rather equivalent and to allocate less, but you are probably more knowledgeable than me